api/docker-compose.yml

```diff
@@ -43,13 +47,15 @@ services:
     environment:
       - 'OPENSEARCH_HOSTS=["http://opensearch-node:9200"]'
       - DISABLE_SECURITY_DASHBOARDS_PLUGIN=true # disables security dashboards plugin in OpenSearch Dashboards
+    networks:
+      - default
 
   localstack:
     container_name: "${LOCALSTACK_DOCKER_NAME:-localstack-main}"
     image: localstack/localstack
     ports:
-      - "127.0.0.1:4566:4566"            # LocalStack Gateway
-      - "127.0.0.1:4510-4559:4510-4559"  # external services port range
+      - "127.0.0.1:4566:4566" # LocalStack Gateway
+      - "127.0.0.1:4510-4559:4510-4559" # external services port range
```

Comment:
Why was this reformatted? I have no preference either way / we have no specific linting for this file, this was just how the formatting came from the OpenSearch docs.

Reply:
That was just my VS code autoformatting for YML files, I think, not something intentional. It looks like the `<code> # <comment>` pattern is more widely used in our codebase than the `<code> #    <comment>` pattern so I'm inclined to keep it this way, but let me know if you'd like me to revert